### ISSUE TITLE
Living up to my username: Total Noob

### ISSUE THREAD

**Issue by Total-Noob1** [coolshou/screenplay-dx#1]
Hi coolshou-
I have an Iomega Screenshare DX that seems to have lost it's 1TB internal hard-drive.  All attempts to update the firmware from within the native interface on the device have failed.  I've searched and searched for help, and this depository of yours is the only one I've found that might be able to help.

I apologize for my ignorance, but I've read your ReadMe file and I still don't understand how to "install" the firmware you've created for the device.  Could you possibly dumb-down the process for me?  i.e.
(1) download the .zip file and extract the contents to an external HDD and connect the drive to the Screenplay DX
(2) turn on the Screenplay DX
(3) then....?

I would greatly appreciate any help you can offer me!

**Comment by coolshou** [comment]
Not sure what you mean "lost it's 1TB internal hard-drive"? Does the internal HDD die? 
Just remove screw at the bottom of DX and open the DX's cover. you may change another new SATA HDD if the old one is not working.

Update Firmware procedure
1. download iomx2.bin , iomx2.fff, iomx2.ver from https://github.com/coolshou/screenplay-dx/releases 
2. place these three files at the root of USB thumb disk
3. plugin USB thumb disk to DX's USB port
4. power on DX, it will try to update firmware if DX's FW version is lower then iomx2.ver 

Using "the native interface" to update FW is not working any more because the official server is down since DX had End Of Life. No more support/update from the Iomega. (bought by Lenovo)

**Comment by Total-Noob1** [comment]
Thanks so much for your help, I'll give these steps a try.
By "lost" the internal hard-drive, I mean that the drive cannot be found.  When using the native interface (on a TV screen/ navigating via the remote control) to access the media server, it says "No Server Found."  When plugging the device into a computer via USB, it shows under the attached devices as a media server, but the storage doesn't mount.  Essentially, it's as if the HDD isn't there.
It's definitely possible that the internal HDD had some sort of mechanical failure, but I'm trying to rule-out any potential software-related issues before physically opening the device...  I looked (pretty closely) and didn't see any screws at all.  I was thinking that the enclosure might be a molded plastic with clips that will need to be stretched out.  I'll look again more closely for a screw on the bottom.
Thanks again for your help, it is greatly appreciated!

Date: Thu, 5 May 2016 03:24:48 -0700

**Comment by coolshou** [comment]
screws are hide under the Rubber mat

**Comment by Total-Noob1** [comment]
Thanks again for your help.  I found the screws - one each under the 4 rubber feet on the bottom.  I've decided that there must be something physically wrong with the circuitry on my ScreenPlay.  Here are the steps I followed:
(1) Updated/ Reinstalled the firmware - your version was the same version as what I already had installed, so it didn't update automatically.  However, I increased the version number on your copy to trick my device into loading your firmware.  It went through the update procedure without any errors, but still showed as having "No Server"(2) Opened the device and installed an alternate SATA HDD that I knew to be functional.  No change - the device still said "No Server" when looking for local files(3) Loaded the HDD from the ScreenPlay into a USB HDD Enclosure and it mounted fine via USB on both the USB port on my computer AND the USB port on the ScreenPlay.  All the previously loaded media files, etc. were there and intact.
In the end, I've confirmed that there's no issue with the HDD, and there's no issue with the firmware, so it must be some sort of physical/ mechanical issue with the circuitry of the ScreenPlay unit itself.  It seems odd, but I can think of no other explanation.
Anyway, thanks again for your help.  I just wanted to give you an update on my findings...

Date: Mon, 9 May 2016 02:37:38 -0700
